src/Dice.hpp

```diff
@@ -0,0 +1,19 @@
+#pragma once
+
+class Dice
+{
+  private:
+    int value;
+public:
+    Dice() = default;
+    ~Dice() = default;
+    Dice(const Dice&) = delete;
+    Dice(Dice&&) = delete;
+    Dice& operator=(const Dice&) = delete;
+    Dice& operator=(Dice&&) = delete;
+
+    void throwDice() noexcept;
```

Comment:
Wrong interface. It is not resistant to use it incorrectly.
Someone can call just getValue() without calling throwDice().
All initialization should take place in constructor - random number generator should go there. throwDice() function should be the only available method and it should return a number from 1 to 12.
All initialization should take place in a constructor.

BTW think if the class is needed for this, maybe only a function should be here? Class should hold some state. I'm not saying that it is not needed. It may be easier to test whole game if we could mock throwing a dice, so it might be useful.